Task5/src/controller/user.controller.ts

```diff
@@ -11,22 +12,22 @@ export class UserController {
 
     public getAllUsers = (req: Request, res: Response) => {
         this.userService.getAllUsers()
-            .then((appusers: any) => res.status(200).send(appusers))
-            .catch((error: any) => res.status(400).send(error))
+            .then((appusers: any) => constructResponse(res, appusers, '', 200))
+            .catch((error: any) => contructErrorResponse(req, res, error, 400, false))
```

Comment:
In case there is a server error, code should be `500` not `400`

Reply:
this is coming from service which also handled by `try catch` block. `500` is handled in globally in `aap.ts`.

Reply:
Yeah, it is handled in service but why use `400` here ?

Reply:
Changed to 500